app/views/providers/providers/show.html.erb

```diff
@@ -14,7 +14,7 @@
       </tr>
       <tr class="govuk-table__row">
         <th class="govuk-table__header" scope="row"><%= t('.role') %></th>
-        <td class="govuk-table__cell"><%= @provider.roles %></td>
+        <td class="govuk-table__cell wrap-text"><%= @provider.roles.gsub(/,(?![ ])/, ', ') %></td>
```

Comment:
I'd suggest this match as being clearer: ,(?=\S)
```ruby
@provider.roles.gsub(/,(?=\S)/, ', ')
```
With `[ ]` it's not really clear what's between the brackets. `\S` is more explicit (anything that is not white space)